### ISSUE TITLE
Parser function breaks in 5.3.4.6

### ISSUE THREAD

**Issue by krabina** [gesinn-it-pub/mediawiki-extensions-PageForms#36]
This field definition showing a set of subpages works in 5.3.4.4 (301aad4), but breaks in 5.3.4.6 (80f19f8) 
{{{field|Seite|input type=combobox|values={{#arraymap:{{SUBPAGES}}|,|@@@@|{{#explode:@@@@|{{FULLPAGENAME}}/|1}}|;}}|delimiter=;}}}

![image](https://user-images.githubusercontent.com/4318745/229234639-5fa7c1b0-4876-4191-8792-03393a60eeb1.png)

**Comment by gesinn-it-gea** [comment]
Looking at the screenshots: both are in multi-instance templates, right? Are both screenshots taken with 5.3.4.6?

**Comment by krabina** [comment]
yes. and both show the problem (double dropdown-arrow), if you try to click, the above picture is happening.

**Comment by krabina** [comment]
Fixed in current master, thank you!

**Comment by krabina** [comment]
Not quite: works for new fields in mutiple instance template, but not for ones holding existing values.
